### DISCUSSION TITLE
blog/acclamato-in-madrepatria

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#5] [Q&A]
# blog/acclamato-in-madrepatria

Ad un anno dal comunicato di UNIBA, mamma Gravina finalmente si è accorta di me. Meglio tardi che mai, mammina ❤️

https://www.pompelmo.xyz/blog/acclamato-in-madrepatria

**Comment by ProtaDec** [comment]
Leggere questo post di prima mattina, mentre pulisco i miei pezzetti di metallo che non sono stati fatti bene forse perché non sono abbastanza "importante", mi dà speranza.

Meriti tutto il riconoscimento che puoi avere!!
